practitioner-service/src/main/java/uk/ac/ox/ndph/mts/practitioner_service/exception/BadRequestException.java

```diff
@@ -0,0 +1,12 @@
+package uk.ac.ox.ndph.mts.practitioner_service.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.BAD_REQUEST)
```

Comment:
Can just use Spring ResponseStatusException instead of a custom class.

Reply:
deleted.